src/Resources/skeleton/user/UserProvider.php.txt

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace App\Security\User;
+
+use App\Entity\User;
+use App\Repository\UserRepository;
+use Doctrine\ORM\NoResultException;
+use Symfony\Component\Security\Core\Exception\UnsupportedUserException;
+use Symfony\Component\Security\Core\Exception\UsernameNotFoundException;
+use Symfony\Component\Security\Core\User\UserInterface;
+use Symfony\Component\Security\Core\User\UserProviderInterface;
+
+class UserProvider implements UserProviderInterface
```

Comment:
If we know the user wants an "entity" provider, then does it make sense to generate this class? They could just use the built-in `entity` provider.